### PR TITLE
Added node for Watson Conversation (Experimental)

### PR DESCRIPTION
**Pull request by ylecleach**

### CODE REVIEW

File: services/conversation/v1-exp.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  function verifyPayload(node, msg) {
+    if (!msg.payload) {
+      this.status({fill:'red', shape:'ring', text:'missing payload'}); 
+      node.error('Missing property: msg.payload', msg);
+      return false;
+    }
+    return true;
+  }
+
+  function verifyInputs(node, msg, config) {
```
Comment (yrezgui) [inline services/conversation/v1-exp.js:42]: Codacy Issue found: [Function 'verifyInputs' has a complexity of 5.](https://www.codacy.com/app/node-red-nodes-watson/node-red-node-watson/file/2780588330/issues/source?bid=3225115&fileBranchId=3365376#l42)

File: services/conversation/v1-exp.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+      return true;
+    } else if (!msg.params || '' !== msg.params.workspace_id) {
+      node.workspaceid = msg.params.workspace_id;
+      return true;
+    }
+    node.error('Missing workspace_id. check node documentation.',msg);
+    return false;
+  }
+
+  function verifyServiceCredentials(node, msg) {
```
Comment (yrezgui) [inline services/conversation/v1-exp.js:56]: Codacy Issue found: [Function 'verifyServiceCredentials' has a complexity of 5.](https://www.codacy.com/app/node-red-nodes-watson/node-red-node-watson/file/2780588330/issues/source?bid=3225115&fileBranchId=3365376#l56)

File: services/conversation/v1-exp.js [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+    return true;
+  }
+
+  function verifyInputs(node, msg, config) {
+    // workspaceid can be either configured in the node, 
+    // or sent into msg.params.workspace_id
+    if (!config.workspaceid || config.workspaceid !== '') {
+      node.workspaceid = config.workspaceid;
+      return true;
+    } else if (!msg.params || '' !== msg.params.workspace_id) {
```
Comment (yrezgui) [inline services/conversation/v1-exp.js]: Codacy Issue found: [Expected literal to be on the right side of !==.](https://www.codacy.com/app/node-red-nodes-watson/node-red-node-watson/file/2780588330/issues/source?bid=3225115&fileBranchId=3365376#l48)

File: services/conversation/v1-exp.js [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+    return true;
+  }
+
+  function verifyInputs(node, msg, config) {
+    // workspaceid can be either configured in the node, 
+    // or sent into msg.params.workspace_id
+    if (!config.workspaceid || config.workspaceid !== '') {
+      node.workspaceid = config.workspaceid;
+      return true;
+    } else if (!msg.params || msg.params.workspace_id !== '') {
```
Comment (chughts) [inline services/conversation/v1-exp.js:48]: Codacy Issue found: [Identifier 'workspace_id' is not in camel case.](https://www.codacy.com/app/BetaWorks-NodeRED-Watson/node-red-node-watson/file/2780632008/issues/source?bid=3330694&fileBranchId=3365344#l48)

**Pull request merged**